examples/highlevel/suite1/save.toml

```diff
@@ -14,7 +15,7 @@ warningTextHasColumn = true
 warningTextHasLine = true
 
 [diff]
-exec_cmd="./ktlint -R diktat-0.4.2.jar -F"
+exec_flags="./ktlint -R diktat-0.4.2.jar -F"
```

Comment:
This was an invalid example, we can't use different binaries in the same suite. Please remove ktlint from here; to keep example complete, use autofix mode of detekt